designing-reports/report-designer-tools/web-report-designer/report-templates.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Report Templates
+page_title: Report Templates in Web Report Designer
+description: Learn more about Report Templates in the Telerik Reporting Web Report Designer, how to create and reuse them.
+slug: web-report-designer-report-templates
+tags: web, report, designer, wrd, report, template, trtx
+published: True 
+position: 1 
+---
+
+<style>
+img[alt$="><"] {
+  border: 1px solid lightgrey;
+}
+</style>
+
+# Report Templates in Web Report Designer
+
+|Minimum Version:|Q4 2025|
+|----|----|
+
+The main purpose of using report templates in the Telerik Web Report Designer is to help users quickly create consistent, professional-looking reports without starting from scratch every time. 
```

Comment:
The first question I would ask: What are templates? And I'd like to see the answer in the intro of the article.